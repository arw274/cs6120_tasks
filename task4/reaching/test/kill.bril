# Block 0 reaching definitions:
# Block 1 reaching definitions: 
#   From block 0 : var a instr 1

@main {
    a: int = const 0;
    a: int = const 1;
    jmp .end;
.end:
    ret;
}