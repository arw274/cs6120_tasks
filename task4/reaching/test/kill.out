Function: main

Block 0: [{'dest': 'a', 'op': 'const', 'type': 'int', 'value': 0}, {'dest': 'a', 'op': 'const', 'type': 'int', 'value': 1}, {'labels': ['end'], 'op': 'jmp'}]
Block 1: [{'label': 'end'}, {'op': 'ret'}]
Block 0 reaching definitions:
Block end reaching definitions:
  From block 0 : var a instr 1

